src/plcc/load_spec/validate_spec/validate_semantic_spec/validate_semantic_spec.py

```diff
@@ -1,13 +1,9 @@
 from dataclasses import dataclass
 from ...load_rough_spec.parse_lines import Line, parse_lines
 from ...parse_spec.parse_semantic_spec import SemanticSpec, CodeFragment
+from .semantic_errors import InvalidClassNameError, UndefinedBlockError, UndefinedTargetLocatorError
```

Comment:
Let's rename `semantic_errors` to `errors`. Because it is in `validate_semantic_spec`, its context is clear. And a "semantic error" should be a mistake in the meaning of something. And this is an error in the specification of semantics, but not necessarily a mistake in its meaning. Subtle. But when I see `semantic_errors` it means something else to me---it IS a semantic error, how ironic!